views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+                        </td>
+                      </tr>
+                      <tr>
+                        <th>sort_name</th>
+                        <td>name for sort ordering: <code>Example, Anne</code></td>
+                      </tr>
+                      <tr>
+                        <th>email</th>
+                        <td>email address: <code>anne@example.com</code></td>
+                      </tr>
+                      <tr>
+                        <th>twitter</th>
+                        <td>twitter handle: <code>example</code> (for <code>@example</code>)</td>
+                      </tr>
+                      <tr>
+                        <th>facebook</th>
+                        <td>Facebook name: <code>AnneExample</code> (for <code>facebook.com/AnneExample)</code>
+                      </tr>
+                      <tr>
+                        <th>group</th>
+                        <td>name of political group <code>Example Faction</code></td>
+                      </tr>
+                      <tr>
+                        <th>group_id<span class="superscript">&dagger;</span></th>
+                        <td>id for group</td>
+                      </tr>
+                      <tr>
+                        <th>area</th>
+                        <td>name of area represented <code>Example County</code></td>
+                      </tr>
+                      <tr>
+                        <th>area_id<span class="superscript">&dagger;</span></th>
+                        <td>id for area</td>
+                      </tr>
+                      <tr>
+                        <th>term<span class="superscript">&dagger;</span></th>
+                        <td>id or index for term</td>
+                      </tr>
+                      <tr>
+                        <th>start_date</th>
+                        <td>start of membership (if needed) <code>2010-11-29</code></td>
+                      </tr>
+                      <tr>
+                        <th>end_date</th>
+                        <td>end of membership (if needed) <code>2012-07-14</code></td>
+                      </tr>
+                      <tr>
+                        <th>image</th>
+                        <td>URL to image file <code>http://example.com/example.jpg</code></td>
+                      </tr>
+                      <tr>
+                        <th>gender</th>
+                        <td>gender if known <code>M</code> or <code>F, or <code>male</code> or <code>female</code></td>
+                      </tr>
+                    </table>
+                  </div>
+                  </li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <p>
+          <span class="superscript">&dagger;</span>
+          You can use fields marked &dagger; as IDs, although we can’t
+          guarantee they’ll always be unique <em>outside</em> this legislature.
```

Comment:
"Can't guarantee" may be a little too weak here. We can almost guarantee that they _won't_ be: lots are just things like `party_id: 1`. It's also a little odd that `id` itself isn't marked with a dagger (it's unclear whether it should be, or whether this description needs to be adjusted slightly)